src/pages/community/members.js

```diff
@@ -10,37 +10,43 @@ import Layout from "../../components/layout";
 import AllMembers from "../../sections/Members-grid/AllMembers";
 import ActiveMembers from "../../sections/Members-grid/ActiveMembers";
 import InactiveMembers from "../../sections/Members-grid/InactiveMembers";
+import Maintainers from "../../sections/Members-grid/Maintainers";
+
 
 const MembersPage = () => {
-    const [active, setActive] = useState(false);
-    const [inActive, setinActive] = useState(false);
 
-    const toggleActive = () => {
-        let status = active;
-        setActive(!status);
-        setinActive(false);
-    };
+    const options = [
+        { label: "All Members", value: "all" },
+        { label: "Active Members", value: "active" },
+        { label: "Inactive Members", value: "inactive" },
+        { label: "Maintainers", value: "maintainers" }
+    ];
 
-    const toggleInactive = () => {
-        let status = inActive;
-        setActive(false);
-        setinActive(!status);
+    const [state, setState] = React.useState({ 
```

Comment:
```suggestion
    const [state, setState] = useState({ 
```

Reply:
Can you use `members` and `setMembers`. This is more appropriate.

Reply:
Okay